### PR TITLE
Add r-parglm

### PR DESCRIPTION
**Pull request by cbrueffer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cbrueffer** [comment]
@mfansler any idea here? The is `m2w64-libwinpthread-git`, but not sure that's the correct one to depend on.

```
number of files: 26
   INFO: sysroot: 'C:/Windows/' files: '['winhlp32.exe', 'win.ini', 'twain_32/wiatwain.ds', 'twain_32.dll']'
   INFO (r-parglm,lib/R/library/parglm/libs/x64/parglm.dll): Needed DSO C:/Windows/System32/KERNEL32.dll found in $SYSROOT
   INFO (r-parglm,lib/R/library/parglm/libs/x64/parglm.dll): Needed DSO C:/Windows/System32/msvcrt.dll found in $SYSROOT
  ERROR (r-parglm,lib/R/library/parglm/libs/x64/parglm.dll): Needed DSO Library/mingw-w64/bin/libwinpthread-1.dll found in ['conda-forge/win-64::m2w64-libwinpthread-git==5.0.0.4634.697f757=2']
  ERROR (r-parglm,lib/R/library/parglm/libs/x64/parglm.dll): .. but ['conda-forge/win-64::m2w64-libwinpthread-git==5.0.0.4634.697f757=2'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (r-parglm,lib/R/library/parglm/libs/x64/parglm.dll): Needed DSO Library/mingw-w64/bin/libgcc_s_seh-1.dll found in conda-forge/win-64::m2w64-gcc-libs-core==5.3.0=7
   INFO (r-parglm,lib/R/library/parglm/libs/x64/parglm.dll): Needed DSO Library/mingw-w64/bin/libstdc++-6.dll found in conda-forge/win-64::m2w64-gcc-libs==5.3.0=7
   INFO (r-parglm,lib/R/library/parglm/libs/x64/parglm.dll): Needed DSO $RPATH/R.dll found in the whitelist
   INFO (r-parglm,lib/R/library/parglm/libs/x64/parglm.dll): Needed DSO $RPATH/Rblas.dll found in the whitelist
   INFO (r-parglm,lib/R/library/parglm/libs/x64/parglm.dll): Needed DSO $RPATH/Rlapack.dll found in the whitelist
```

**Comment by mfansler** [comment]
Yeah, you can use `- {{ native }} libwinpthread-git` for now, just to get past the overlinking error, but you'll have to remove it later when the UCRT migrator sends a PR.

File: recipes/r-parglm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ stdlib("c") }}
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - {{ native }}libwinpthread-git  # [win]
```
Comment (mfansler) [inline recipes/r-parglm/meta.yaml:39]: Should be in run.

**Comment by cbrueffer** [comment]
Thanks for the help @mfansler!

**Pull request merged**